tfjs-backend-webgpu/src/kernels/matmul_packed_webgpu.ts

```diff
@@ -121,6 +121,63 @@ export function makeMatMulPackedSource(workPerThread: number[]): string {
   `;
 }
 
+export function makeMatMulVectorSource(): string {
+  return `
+    float mm_readA(int row, int col);
+    float mm_readB(int row, int col);
+    void mm_write(int row, int col, float value);
+    void mm_matMul(int dimAOuter, int dimInner, int dimBOuter);
+
+    const int TileSize = int(gl_WorkGroupSize.x) * 4;
+
+    shared vec4 mm_Asub[TileSize / 4];
+
+    void mm_matMul(int dimAOuter, int dimInner, int dimBOuter) {
+      int tileCol = int(gl_LocalInvocationID.x);
+      int globalCol = int(gl_GlobalInvocationID.x);
+      int globalRow = int(gl_GlobalInvocationID.y);
+
+      int numTiles = (dimInner - 1) / TileSize + 1;
+
+      float acc;
+      vec4 ACached;
+      vec4 BCached;
+
+      // Without this initialization strange values show up in acc.
+      acc = 0.0;
```

Comment:
nit: Can be on the same line with the declaration

Reply:
Done